webapp/i18n/en.json

```diff
@@ -138,6 +148,20 @@
   "Mutator.new-card-from-template": "new card from template",
   "Mutator.new-template-from-board": "new template from board",
   "Mutator.new-template-from-card": "new template from card",
+  "OnboardingTour.AddComments.Body": "You can comment on issues, and even @mention your fellow Mattermost users to get their attention.",
```

Comment:
These strings shouldn't be updated as part of this PR, right?

Reply:
I think the diff is unfortunately a bit misleading—it looks like a few sections got re-ordered.

I found when working through Scott's feedback that the `18n-extract` command was a bit fragile in how it pattern matched the files to process, so I fixed that and re-ran it in [`941994f` (#3071)](https://github.com/mattermost/focalboard/pull/3071/commits/941994f829396a215065576cb85b018705befb5d)